Orange/widgets/unsupervised/owdistances.py

```diff
@@ -48,6 +48,8 @@ class Error(OWWidget.Error):
         dense_metric_sparse_data = Msg("Selected metric does not support sparse data")
         empty_data = Msg("Empty data set")
         mahalanobis_error = Msg("{}")
+        distances_memory_error = Msg("Not enough memory.")
+        distances_value_error = Msg("Error occurred while calculating distances: {}")
```

Comment:
The error can be long, so `"Error occurred while calculating distances\n{}"`. Skip the colon, so the error looks OK, and the tooltip shows the entire message.